sql/parse/parse.go

```diff
@@ -173,6 +147,22 @@ func convert(ctx *sql.Context, stmt sqlparser.Statement, query string) (sql.Node
 	}
 }
 
+func convertExplain(ctx *sql.Context, n *sqlparser.Explain) (sql.Node, error) {
+	child, err := convert(ctx, n.Statement, "")
+	if err != nil {
+		return nil, err
+	}
+
+	if n.ExplainFormat != "" && strings.ToLower(n.ExplainFormat) != sqlparser.TreeStr {
```

Comment:
Right now we only support `tree`, but would it be better for this to be a loop or some kind of `array.Contains` over all of the supported `sqlparser` formats? Or perhaps a switch statement with the default returning this error?

Reply:
Changed to a switch.